### PR TITLE
Update to v1.1.1

### PR DESCRIPTION
**Pull request by rmrt1n**
Hi I tried updating the app version

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build

**Review by dschier-wtd — APPROVED** [review]
- code: lgtm (with one minor improvement)
- build: worked
- test: runs fine on f34,f35,c9

Thanks a lot for the contribution, I really appreciate it.

File: com.shatteredpixel.shatteredpixeldungeon.appdata.xml [hunk 1]
```diff
@@ -42,6 +42,11 @@
 		</screenshot>
 	</screenshots>
 	<releases>
+		<release version="1.1.1" date="2022-01-04">
+			<description>
+				<p>v1.1 contains various improvements to the alchemy system and the items it produces, and a music track for each region of the game! There are a few new spells, and several buffs and reworks to exotic potions and scrolls in particular, and various other smaller changes.</p>
```
[the remark this answers is not in the document]
Comment (rmrt1n) [reply com.shatteredpixel.shatteredpixeldungeon.appdata.xml]: i have updated it

**Comment by dschier-wtd** [comment]
Thanks a lot for your contribution. :+1:

**Pull request merged**